### PR TITLE
Release F branch PR

### PR DESCRIPTION
**Pull request by willh**
* Profile pages improvements
- Added lazy load approach to fetching civilServant entity (with custom hql queries utilizing lazy fetch)
- added userId (for usage on lpg-ui)
- added an optional accessLog filter for logging request details

* Restored eager loading of children entitities for now

Co-authored-by: Michal Piatek <m.piatel@kainos.com>
Co-authored-by: Matt Doherty <61981603+mattdoherty-kainos@users.noreply.github.com>

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -30,6 +31,7 @@ spring:
 
 server:
   port: 9002
+  requestLogginFilterEnabled: ${REQUEST_LOGGING_FILTER_ENABLED:false}
```
Comment (mmackiewicz1) [inline src/main/resources/application.yml:34]: We need to remember to have this configuration set to false, so we will not log every request.

Comment (willh) [reply src/main/resources/application.yml:34]: Yeah definitely one for debugging only.

File: src/main/java/uk/gov/cshr/civilservant/filter/AccessLogFilter.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.Enumeration;
+
+@Component
+public class AccessLogFilter implements Filter {
```
Comment (mmackiewicz1) [inline src/main/java/uk/gov/cshr/civilservant/filter/AccessLogFilter.java:19]: Useful for debugging requests.

**Comment by willh** [comment]
Build connector from Github side seems to have timed out... need to push a readme change to trigger branch build

**Pull request merged**